[2020-06-24 19:06:54,752] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: integrator.task_id_1 2020-06-21T00:00:00+00:00 [queued]>
[2020-06-24 19:06:54,803] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: integrator.task_id_1 2020-06-21T00:00:00+00:00 [queued]>
[2020-06-24 19:06:54,804] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-24 19:06:54,804] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-24 19:06:54,804] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-24 19:06:54,849] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): task_id_1> on 2020-06-21T00:00:00+00:00
[2020-06-24 19:06:54,853] {standard_task_runner.py:53} INFO - Started process 7728 to run task
[2020-06-24 19:06:54,989] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: integrator.task_id_1 2020-06-21T00:00:00+00:00 [running]> strx
[2020-06-24 19:06:55,070] {logging_mixin.py:112} INFO - hello
[2020-06-24 19:06:55,071] {taskinstance.py:1145} ERROR - name 'MySqlHook' is not defined
Traceback (most recent call last):
  File "/home/isaias/anaconda3/envs/cmpc/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 983, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/isaias/anaconda3/envs/cmpc/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/home/isaias/anaconda3/envs/cmpc/lib/python3.6/site-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/isaias/airflow/dags/integrator.py", line 108, in HelloDBOperator
    hook = MongoHook(conn_id=mongo_conn_id)
NameError: name 'MySqlHook' is not defined
[2020-06-24 19:06:55,073] {taskinstance.py:1168} INFO - Marking task as UP_FOR_RETRY
[2020-06-24 19:07:04,705] {logging_mixin.py:112} INFO - [2020-06-24 19:07:04,704] {local_task_job.py:103} INFO - Task exited with return code 1
